_mentalModels/2022-03-20-stairs.md

```diff
@@ -0,0 +1,50 @@
+---
+title: The Stairs
+
+draft: false
+
+page : false 
+
+image: /assets/img/mental-models/stairs.svg
+
+excerpt: >
+  The Stairs is a good metaphor to think about _progression_- specially with a human factor.
+
+---
+
+<img
+  alt="A schematic stairs pattern"
+  src="/assets/img/mental-models/stairs.svg"
+  style="max-height: 480px; margin: 2rem auto"
+/>
+
+The Stairs is a good metaphor to think about _progression_- specially with a
+human factor.
+
+It could be about how a team project is going at work. Or the typical learning
+curve of mastering a new language. Or a personal advancement toward a career
+goal. This is never linear.
```

Comment:
```suggestion
It could be about the way a team project is going at work, or the typical learning
curve of mastering a new language. It can even be about one's personal advancement toward a career
goal. It is somewhat linear when zoomed out (there's a starting line and a goal), but if you zoom in, you'll find that it's not a straight line.
```